### DISCUSSION TITLE
关于用户界面的视频

### DISCUSSION THREAD

**Discussion by Shaddoll126** [ywmoyue/biliuwp-lite#244] [Q&A]
视频右下角的数字是什么意思啊OAO
![屏幕截图 2023-08-04 173833](https://github.com/ywmoyue/biliuwp-lite/assets/124747209/4390cccb-7383-4ed0-b9a9-147aad69a6b7)

**Comment by ywmoyue** [comment]
视频时长，问下这是哪个页面的，我处理下

**Reply by Shaddoll126** [reply]
就是用户中心界面的投稿视频页面:3
